### ISSUE TITLE
Calculating your Arctos fee

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/documentation-wiki#46]
I have created a spreadsheet for anyone who wants to calculate their Arctos fee.  Any way we can incorporate this into the Wiki?
[Arctos Annual Fee Calculation.xlsx](https://github.com/ArctosDB/documentation-wiki/files/1267569/Arctos.Annual.Fee.Calculation.xlsx)

**Comment by mkoo** [comment]
Should this go in the handbook or here: https://arctosdb.org/join-arctos/costs/

@ccicero @campmlc topic for AWG meeting?

**Comment by campmlc** [comment]
I think the latter. However, I suggest we wait on this until we can discuss
our financial model at the June meeting.
Mariel

**Comment by Jegelewicz** [comment]
I agree it should go on the website. I'd like to see it up even if it is going to change in June, but if it causes a lot of extra effort, let's just wait.

**Comment by campmlc** [comment]
We could put it up, and send out a note to all Arctos collections to review
it and provide comments for the June meeting discussion.

**Comment by jrdemboski** [comment]
Yes, put it up at https://arctosdb.org/join-arctos/costs/, that is our current fee structure even if it might (have to) change after our June discussion

John D

**Comment by mkoo** [comment]
I think it's a cool little calculator and I can post that it's for
generating ESTIMATES ONLY.
Thanks all

**Comment by mkoo** [comment]
I posted an online version of Teresa's calculator with a note that it will be updated in june.
Closing this for now. We can open a new issue when we have a new updated formula.

**Comment by Jegelewicz** [comment]
Thanks!
